src/notation/view/percussionpanel/percussionpanelpadlistmodel.h

```diff
@@ -27,12 +27,14 @@
 #include "async/asyncable.h"
 #include "async/channel.h"
 
+#include "engraving/rendering/engravingitempreviewpainter.h"
 #include "engraving/dom/drumset.h"
 
 #include "percussionpanelpadmodel.h"
 
 static constexpr int NUM_COLUMNS(8);
```

Comment:
Unrelated, but shouldn't this be a (private?) member of `PercussionPanelPadListModel`?